lookout/core/cmdline.py

```diff
@@ -121,6 +123,17 @@ def add_model_repository_args(parser):
                help="Additional keyword arguments to SQLAlchemy database engine.")
 
 
+def tool(args) -> None:
```

Comment:
Can you please rename function tool to run_cmdline_tool in the analyzers (here is fine)